uploader/wiki_uploader.py

```diff
@@ -23,13 +23,14 @@ def __init__(
             access_secret=access_secret,
         )
 
-    def upload_file(self, file_name, file_stream, description=""):
+    def upload_file(self, file_name, file_stream, datecreated="", description=""):
         if not description:
             description = file_name
 
         upload_result = self.mw_client.upload(
             file=file_stream,
             filename=file_name,
+            datecreated=datecreated,
```

Comment:
are you sure this would work ? https://github.com/mwclient/mwclient/blob/master/mwclient/client.py#L615 shows that the function does not expect a parameter `datecreated` Can you paste a screenshot from the wiki after this upload to see if the date created is showing up there correctly ?